app/serializers/job_serializer.rb

```diff
@@ -1,4 +1,5 @@
 class JobSerializer < ActiveModel::Serializer
-  attributes :title, :description, :url, :location, :posted_date, :remote, :raw_technologies
+  attributes :id, :title, :description, :url, :location, :posted_date, :remote
```

Comment:
IMO it makes more sense to be relying on the association with technologies, since raw technologies is, as it says, "raw". So glad you made this happen!

We should follow up though with another issue about whether we need to have an ongoing look at what technologies are included in the Technology table, since that's fixed and technology, as a rule, is not.